broker/src/main/java/org/apache/rocketmq/broker/BrokerController.java

```diff
@@ -650,10 +652,13 @@ public void protectBroker() {
 
     public long headSlowTimeMills(BlockingQueue<Runnable> q) {
         long slowTimeMills = 0;
-        final Runnable peek = q.peek();
-        if (peek != null) {
-            RequestTask rt = BrokerFastFailure.castRunnable(peek);
-            slowTimeMills = rt == null ? 0 : this.messageStore.now() - rt.getCreateTimestamp();
+        Optional<RequestTask> op = q.stream()
+                .map(BrokerFastFailure::castRunnable)
+                .filter(Objects::nonNull)
+                .findFirst();
+        if (op.isPresent()) {
```

Comment:
i wonder if stream is need in this place. null check is much low cost.

Reply:
Thanks for your review.

But I can't catch the point of your question, what's wrong with `stream`?

Reply:
my bad. the code is right after i check the discussion.